### PR TITLE
Aadhar kyc

### PR DESCRIPTION
**Pull request by Amay515**
Created All the screen for Aadhar Kyc

### CODE REVIEW

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+    checkMark: false,
+  });
+  const [kycReqID, setKycReqID] = React.useState("");
+  const [captchaImg, setCaptchaImg] = React.useState("");
+  const [loading, setLoading] = React.useState(true);
+  const [loading2, setLoading2] = React.useState(false);
+
+  React.useEffect(() => {
+    const headers = {
+      "x-client-id": "401c3159-bbfe-438d-a62c-eb4ff8bb8c10",
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:23]: use env file to get client id, secret and base URL

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      "x-client-secret": "b066a8b1-c3f9-432d-ae74-79489047ec87",
+    };
+
+    const getCaptcha = (id) => {
+      const url2 = `https://dg-sandbox.setu.co/api/okyc/${id}/initiate/`;
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:28]: use env file to get  base URL

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+            setCaptchaImg(res.captchaImage);
+          }
+        })
+        .catch((err) => console.log(err))
+        .finally(() => {
+          setLoading(false);
+        });
+    };
+
+    const url1 = "https://dg-sandbox.setu.co/api/okyc/";
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:49]: use env file to get  base URL

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+          localStorage.setItem("KYCreqID", res.id);
+        }
+      })
+      .catch((error) => console.log(error));
+  }, []);
+
+  const sendData = () => {
+    setLoading2(true);
+
+    const url = `https://dg-sandbox.setu.co/api/okyc/${kycReqID}/verify`;
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:71]: use env file to get  base URL

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+    const headers = {
+      "Content-Type": "application/json",
+      "x-client-id": "401c3159-bbfe-438d-a62c-eb4ff8bb8c10",
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:75]: use env file to get client id, secret and base URL

File: apps/front-end/src/routes/admin.js [hunk 6]
```diff
@@ -6,12 +6,35 @@ import FacilitatorForm from "../pages/admin/FacilitatorForm";
...
   { path: "/admin/view/:id", component: FacilitatorView },
   { path: "/admin/facilitator-onbording", component: FacilitatorForm },
   { path: "/admin/profile", component: Profile },
   { path: "/orientation", component: orientationScreen }, 
   { path: "/admin", component: AdminHome },
   { path: "/", component: AdminHome },
-  { path: "*", component: NotFound },
+
+  {
+    path: '/adharStart',
```
Comment (manojLondhe) [inline apps/front-end/src/routes/admin.js:23]: correct spellings everywhere 
adhar => aadhaar

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+    <div className="adharVerify1">
+      <div className="topbar">
+        <button className="btn-back" onClick={() => navigate(-1)}>
+          <WestIcon />
+        </button>
+      </div>
+
+      <div className="content">
+        <h2>
+          Offline Aadhaar Verification
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:114]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+          <br />
+          (OKYC)
+        </h2>
+
+        <div className="aadhaarNumber__input">
+          <label htmlFor="aadhaarNumber">Enter Your Aadhaar Number</label>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:120]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 9]
```diff
@@ -0,0 +1,206 @@
...
+            borderColor={"rgba(0, 0, 0, 0.5)"}
+            onChange={(e) => {
+              setData({ ...data, aadhaarNumber: e.target?.value });
+            }}
+          />
+        </div>
+
+        <div className="securityCode__input">
+          <label htmlFor="securityCode">
+            <span>Enter Security Code</span>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:138]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 10]
```diff
@@ -0,0 +1,206 @@
...
+            borderRadius={8}
+            borderColor={"rgba(0, 0, 0, 0.5)"}
+            onChange={(e) => {
+              setData({ ...data, securityCode: e.target?.value });
+            }}
+          />
+        </div>
+
+        <p className="boxMessage">
+          We will send you an OTP to the mobile number linked with your Aadhaar
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:160]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 11]
```diff
@@ -0,0 +1,206 @@
...
+        </p>
+
+        <ul>
+          <li>
+            I agree to download my Aadhaar XML file from the UIDAI Website to
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:165]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 12]
```diff
@@ -0,0 +1,206 @@
...
+            complete Aadhaar offline verification with smallcase.{" "}
+          </li>
+
+          <li>
+            I understand that my Aadhaar details shall not be used or stored for
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:170]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 13]
```diff
@@ -0,0 +1,206 @@
...
+        <label htmlFor="checkMark">
+          <Checkbox
+            id="checkMark"
+            name="checkMark"
+            size="small"
+            onChange={(e) => {
+              setData({ ...data, checkMark: e.target.checked });
+            }}
+          />{" "}
+          I have read and understood all of the points above
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:184]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js [hunk 14]
```diff
@@ -0,0 +1,206 @@
...
+              : "gray.500"
+          }
+          py="12px"
+          px="20px"
+          disabled={
+            !data.aadhaarNumber || !data.securityCode || !data.checkMark
+          }
+          onPress={sendData}
+        >
+          <Text color="white">Continue</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js:201]: use lang file for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js [hunk 15]
```diff
@@ -0,0 +1,133 @@
...
+      shareCode: data.securityCode,
+      aadhaarNumber,
+    };
+
+    const url = `https://dg-sandbox.setu.co/api/okyc/${kycReqID}/complete/`;
+
+    fetch(url, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js:38]: use env file to get client id, secret and base URL

File: apps/front-end/src/pages/front-end/AadhaarKyc/QrScannerKyc/QrScannerKyc.js [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+  return (
+    <div className="qrScannerPage">
+      <div className="topbar">
+        <button className="btn-back" onClick={() => navigate(-1)}>
+          <WestIcon />
+        </button>
+      </div>
+
+      <div className="content">
+        <h2>Scan Your Aadhaar QR Code</h2>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/QrScannerKyc/QrScannerKyc.js:74]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/QrScannerKyc/QrScannerKyc.js [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+
+        <video ref={videoRef} style={{ width: "100%" }} />
+        <canvas ref={canvasRef} style={{ display: "none" }} />
+
+        <Button variant="secondary" bg={"gray.500"} py="12px" px="20px" mt={10}>
+          <Text color="white">Continue</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/QrScannerKyc/QrScannerKyc.js:80]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 18]
```diff
@@ -0,0 +1,247 @@
...
+      <div className="topbar">
+        <button className="btn-back" onClick={() => navigate(-1)}>
+          <WestIcon />
+        </button>
+      </div>
+
+      <div className="content">
+        {!submitted ? (
+          <>
+            <h2>Aadhaar Card</h2>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:84]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 19]
```diff
@@ -0,0 +1,247 @@
...
+            <Text>Upload a photo or scan of your card</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:85]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 20]
```diff
@@ -0,0 +1,247 @@
...
+                setModal(true);
+              }}
+            >
+              {isFront ? (
+                image.front ? (
+                  <img src={image.front} alt="front image" />
+                ) : (
+                  <>
+                    <UploadIcon />
+                    <Text>Upload the front side of your Aadhaar card</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:99]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 21]
```diff
@@ -0,0 +1,247 @@
...
+                  </>
+                )
+              ) : image.back ? (
+                <img src={image.back} alt="back image" />
+              ) : (
+                <>
+                  <UploadIcon />
+                  <Text>Upload the back side of your Aadhaar card</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:107]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 22]
```diff
@@ -0,0 +1,247 @@
...
+                bg="gray.400"
+                py="12px"
+                px="20px"
+                mt="10"
+                onPress={() => {
+                  setImage((prev) => ({ ...prev, back: "" }));
+                  setModal(true);
+                }}
+              >
+                <Text color="white">Upload Again</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:152]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 23]
```diff
@@ -0,0 +1,247 @@
...
+            <Button
+              variant="secondary"
+              bg={!image.front || !image.back ? "gray.300" : "gray.500"}
+              py="12px"
+              px="20px"
+              mt="10"
+              disabled={!image.front || !image.back}
+              onPress={handleSubmit}
+            >
+              <Text color="white">Continue</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:165]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 24]
```diff
@@ -0,0 +1,247 @@
...
+            </Button>
+          </>
+        ) : (
+          <>
+            <p className="boxMessage3">
+              <CheckIcon fontSize="medium" />
+              <span>Aadhaar Card Uploaded</span>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:172]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 25]
```diff
@@ -0,0 +1,247 @@
...
+            </p>
+
+            <div className="front__image">
+              <h4>Front View</h4>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:176]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js [hunk 26]
```diff
@@ -0,0 +1,247 @@
...
+
+              <img src={image.front} alt="front image" />
+            </div>
+
+            <div className="back__image">
+              <h4>Back View</h4>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/ManualUpload/ManualUpload.js:182]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js [hunk 27]
```diff
@@ -0,0 +1,81 @@
...
+
+      <div className="bottom">
+        <Text textAlign="center" mb="4">We have sent you a text message with username and password on your mobile number</Text>
+        
+        <Button
+          variant="outline"
+          w="100%"
+          borderRadius="full"
+        >
+          Skip to Login
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js:74]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js [hunk 28]
```diff
@@ -0,0 +1,81 @@
...
+          borderRadius="full"
+          mt="8"
+          py="3"
+        >
+          <Text fontSize="md" fontWeight="medium" color="white">Manual Aadhaar Upload</Text>
+        </Button>
+      </div>
+
+      <div className="bottom">
+        <Text textAlign="center" mb="4">We have sent you a text message with username and password on your mobile number</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js:67]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js [hunk 29]
```diff
@@ -0,0 +1,81 @@
...
+import ErrorOutlineIcon from '@mui/icons-material/ErrorOutline';
+
+import "./AadhaarKyc.css";
+import { Button, Text } from "native-base";
+
+const AadhaarStartKyc = () => {
+  return (
+    <div className="adharVerifyStart">
+      <div className="content">
+        <p>Your Application is Submitted Successfully!</p>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js:11]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js [hunk 30]
```diff
@@ -0,0 +1,81 @@
...
+
+        <h1>Complete Aadhaar Card Verification</h1>
+
+        <Button
+          variant="outline"
+          bg="gray.100"
+          borderRadius="full"
+          mt="10"
+        >
+          <Text fontSize="md" fontWeight="medium" color="gray.600">Aadhaar Number KYC</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js:21]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js [hunk 31]
```diff
@@ -0,0 +1,81 @@
...
+
+        <Text
+          color="red.600"
+          display="flex"
+          alignItems="center"
+          gap="1"
+          mt="2"
+        >
+          <ErrorOutlineIcon fontSize="small" />
+          <span style={{ fontSize: "13px" }}>Mobile Number is not linked to Aadhaar Card</span>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadharStartKyc.js:32]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarSuccess.js [hunk 32]
```diff
@@ -0,0 +1,34 @@
...
+    <div className="adharVerify2">
+      <div className="topbar">
+        <button className="btn-back" onClick={() => navigate(-1)}>
+          <WestIcon />
+        </button>
+      </div>
+
+      <div className="content">
+        <h2>
+          Offline Aadhaar Verification
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarSuccess.js:18]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarSuccess.js [hunk 33]
```diff
@@ -0,0 +1,34 @@
...
+          <br />
+          (OKYC)
+        </h2>
+
+        <p className="boxMessage2">
+          <CheckIcon fontSize="small" />
+          <span>Your Aadhaar Verification is successful</span>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarSuccess.js:25]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarSuccess.js [hunk 34]
```diff
@@ -0,0 +1,34 @@
...
+        </p>
+
+        <Button variant="secondary" bg={"gray.500"} py="12px" px="20px" mt={20}>
+          <Text color="white">Continue Login</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarSuccess.js:29]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js [hunk 35]
```diff
@@ -0,0 +1,133 @@
...
+        <Button
+          variant="secondary"
+          bg={!data.otpNumber || !data.securityCode ? "gray.300" : "gray.500"}
+          py="12px"
+          px="20px"
+          mt={20}
+          disabled={!data.otpNumber || !data.securityCode}
+          onPress={handleSubmit}
+        >
+          <Text color="white">Continue</Text>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js:128]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js [hunk 36]
```diff
@@ -0,0 +1,133 @@
...
+              setData({ ...data, otpNumber: e.target?.value });
+            }}
+          />
+        </div>
+
+        <div className="securityCode__input" style={{ marginTop: "30px" }}>
+          <label htmlFor="securityCode">
+            <span>Enter Security Code</span>
+            <span>
+              Set a 4 digit passcode to securely share your Aadhaar zip filee
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js:101]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js [hunk 37]
```diff
@@ -0,0 +1,133 @@
...
+
+      <div className="content">
+        <h2>
+          Offline Aadhaar Verification
+          <br />
+          (OKYC)
+        </h2>
+
+        <div className="aadhaarNumber__input" style={{ marginTop: "40px" }}>
+          <label htmlFor="otpNumber">Enter 6 digit OTP</label>
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js:80]: use lang file and strings for all UI text - add text for both hindi, english

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js [hunk 38]
```diff
@@ -0,0 +1,133 @@
...
+    <div className="adharVerify2">
+      <div className="topbar">
+        <button className="btn-back" onClick={() => navigate(-1)}>
+          <WestIcon />
+        </button>
+      </div>
+
+      <div className="content">
+        <h2>
+          Offline Aadhaar Verification
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarOTP.js:74]: use lang file and strings for all UI text - add text for both hindi, english

**Review by manojLondhe — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js (hunk 5 above)
Comment (Amay515) [reply apps/front-end/src/pages/front-end/AadhaarKyc/AadhaarKyc.js]: done

**Comment by manojLondhe** [comment]
closing in favour of https://github.com/tekdi/eg-website/pull/49

**Pull request closed without merging**